VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleListEntryController.cs

```diff
@@ -217,6 +209,100 @@ public IHttpActionResult Move(webModel.MoveInfo moveInfo)
             return Ok();
         }
 
+        /// <summary>
+        /// Bulk deletes the specified items by ids.
+        /// </summary>
+        /// <param name="searchCriteria"></param>
+        [HttpPost]
+        [Route("delete")]
+        [ResponseType(typeof(void))]
+        public IHttpActionResult BulkDelete(webModel.SearchCriteria searchCriteria)
```

Comment:
I think better to rename this method to Delete (if not exists already) or DeleteListEntries.